terraform/deploy_new_adi_lambda.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# deploy_new_adi_lambda.sh region workspace_name
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [ $# -eq 2 ]
+then
+    region=$1
```

Comment:
I don't think `region` is ever used.  It can be removed.

Reply:
Ha, good catch, thanks! See https://github.com/cisagov/cyhy_amis/pull/246/commits/5a213187f09a1eff98626c295d135fb4ece6b201 for the change.  I was overzealous in my copy/pasting of a similar deploy script.

Reply:
![image](https://user-images.githubusercontent.com/5521725/70267915-6ca75080-176d-11ea-88d1-cd7c747e8f69.png)